CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 This file tracks changes to this project. It follows the [Keep a Changelog format](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - 2025-02-02
+
+Logging and Server request handling revamped to hopefully be more familiar interfaces for users.
+
+### Removed
+
+- Logger and LogLevel enum have been removed.
+
+### Changed
+
+- Server.Logger is now of type `func(r *http.Request) *slog.Logger` instead of `sse.Logger`
+- Server.OnSession signature changed from `func(s *sse.Session) (Subscription, bool)` to `func(w http.ResponseWriter, r
+*http.Request) (topics []string, accepted bool)`
+- complex example updated to demonstrate new behavior
```

Comment:
That's technically not a change to the library itself so let's not cover it in the changelog.
```suggestion
```